spyderlib/config.py

```diff
@@ -513,7 +513,7 @@ def is_ubuntu():
               # -- In widgets/editor
               'editor/inspect current object': 'Ctrl+I',
               'editor/go to line': 'Ctrl+L',
-              'editor/file list management': 'Ctrl+E',

```

Comment:
Since you're removing this config option, you need to bump `CONF_VERSION` below (in this same file, I mean :-)